.github/workflows/ci.yml

```diff
@@ -12,6 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-20.04
```

Comment:
TODO: In next ci enhancement PR, we could try ubuntu-latest or at least a new LTS version to avoid running in outdated env